provisioning/group_vars/all.yml

```diff
@@ -25,6 +25,12 @@ apt_packages:
   - sqlite3
   - zlib1g-dev
 
+bash_alias_lines:
+  - "alias cdmorph='if [ \"$USER\" = \"deploy\" ]; then cd /var/www/current; export RAILS_ENV=production; env | egrep RAILS_ENV; else cd /vagrant ; echo \"development. sudo -u deploy -i for production!\"; fi'"
+  - 'alias be="bundle exec"'
+  - 'alias r="bundle exec rails"'
+  - 'alias rk="bundle exec rake"'
+
```

Comment:
Is that were we install on prod?  (okay, I  checked it is)
We're using `/srv/` on other deploys.

Reply:
these do look like personal aliases, but I can't see the harm in having them.